src/main/java/com/headtrixz/ui/Home.java

```diff
@@ -20,4 +26,18 @@ public class Home {
     public void playTicTacToe() throws Exception {
         UIManager.switchScreen("game-mode");
     }
+
+    public void canPlay(Boolean bool){
+        playTicTacToeButton.setDisable(!bool);
+    }
+
+    public void connect() {
+        UIManager.setSetting("username", username.getText());
+        UIManager.setSetting("address", address.getText());
+        UIManager.setSetting("port", port.getText());
+
+        canPlay(true);
```

Comment:
Username is inderdaad nodig wanneer je wilt spelen, alleen om tegen de ai te spelen hoef je niet perse online te zijn.

Reply:
Ohja dit was ik inderdaad vergeten oops. Is het dan handig om nog een knop te maken waarbij de usrname wordt opgeslagen?

Reply:
Ja, verplaats anders de username op de ui naar boven de "online settings text" en maak een apparte method om de username op te slaan.
Ook wanneer er een connectie is gemaakt alle 3 velden disablen.